src/components/left-section/LeftSection.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+export function LeftSection({ addPostShowModal, showModal, setShowModal }) {
```

Comment:
suggestion : folder name can also have name without hyphen.
left section seems a side bar navigation so name can be improved to sidebar.

Reply:
Done